skawa_components/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.5.1
```

Comment:
Also modify the pubspec.yaml to the coreesponding version

Reply:
Done.